src/components/CommentMessage/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { CSSTransition } from 'react-transition-group';
+
+interface CommentMessageProps {
+  message: string;
+}
+
+export default function CommentMessage({
+  message
+}: CommentMessageProps) {
+  return (
+    <CSSTransition
+      in={message !== ''}
+      timeout={400}
+      classNames="comment-message-"
+      unmountOnExit
+    >
+      <div className="comment-message" role="alert">
```

Comment:
The role looks odd to me, would be very distracting to users to have all of these read out loud when the page loads.

```suggestion
      <div className="comment-message">
```

Reply:
These should be flash messages, flashing up briefly when a saved comment is edited/deleted/etc - my concern was making sure screenreader users got the save warning directly after the action. They shouldn't be showing up on load...

Reply:
Do you have any suggestions for how I should do this?

Reply:
Oh ok, I think I likely misunderstood this. Will be easier when I try it out properly.

Reply:
You’re right, this is great as-is!